config/ingest.py

```diff
@@ -0,0 +1,12 @@
+# Config override for lsst.pipe.tasks.IngestTask
+config.parse.translation = {'visit': 'OBSID',
+                            'expTime': 'EXPTIME',
```

Comment:
Why is `"expTime": "EXPTIME"` necessary? Surely header name lookup is case blind?

Reply:
Columns must be present in `translation` or `translators` to be read at all. I tried testing the keywords just now and they are, surprisingly, case-sensitive.

(Technically I _don't_ need the exposure time, but it's one of the few lsstSim columns that can be read without special code support so I figured I might as well leave it in.)